src/MovieDetails/MovieDetails.css

```diff
@@ -13,26 +13,60 @@
   position: relative;
 }
 
-/* .movie-info {
+.movie-info {
   position: absolute;
-  width: 250px;
+  /* width: 250px; */
   left: 6em;
   top: 20em;
   display: flex;
-} */
+}
+
+.movie-aside {
+  margin: 1em 1em 1em 4em;
+}
 
 .poster-img {
-  position: absolute;
+  /* position: absolute; */
   width: 250px;
-  left: 6em;
-  top: 20em;
+  margin-bottom: 1.75em;
+  /* left: 6em; */
+  /* top: 20em; */
 }
 
 .movie-main {
   color: white;
-  position: absolute;
-  left: 25em;
-  top: 17em;
+  margin-left: 2em;
+  margin-right: 10em;
+  width: 75%;
+  /* position: absolute; */
+  /* left: 25em;
+  /* top: 17em; */
```

Comment:
I know we are still working on responsiveness, but we need to remember to clean up these styles once we figure that out.